pages.pl/common/[.md

```diff
@@ -0,0 +1,25 @@
+# [
+
+> Porównuje wartości.
```

Comment:
gTranslate translates this as `Compare the values`. I don't speak the language, but the `[` doesn't compare values.
The `==` compares values, `-z` or `-f` evaluate a specific characteristic, e.g. `it's an empty variable, `the file exists`, etc.
Could you check this translation from `Evaluate condition.`?

Reply:
You are correct, I might have shortened it a bit too much, but in my defence the [English TLDR page](https://github.com/tldr-pages/tldr/blob/5ff5ff69e266952d82fa2a69024b3f5b296e1eb3/pages/common/test.md) says:
> Evaluate condition. Returns 0 if the condition evaluates to true, 1 if it evaluates to false.
> More information: https://www.gnu.org/software/coreutils/test.

***

The manpage of test says:
> check file types and compare values

And PL manpage:
> sprawdza typy plików i porównuje wartości

Which translates to roughly the same as the EN manpage

***

How about I change both EN and PL pages to their manpage equivalents? The English "Evaluate condition" while a bit more valid than my "Porównuje wartości" (as it denotes any logic test and my suggest a two operand comparasion) still is a bit to vague and this might be TL;DR but it is too much.

Updated pull request with the proposed PL and EN changes.

Reply:
The description in the man page is a bit misleading. Since our aim is to help new users, we should be more descriptive here.

Reply:
How about:

EN: Compares strings, integers. Checks and compares file attributes. Performs boolean operations.  
(PL: Porównuje ciągi znaków, liczby całkowite. Sprawdza i porównuje atrybuty plików. Wykonuje operacje logiczne.)

This does not 100% describe:

- `-n`, `-z` - but they are shorthands of `$str != ""` and `$str = ""`
- `-d`, `-e`, `-f` - it says "checks (...) file attributes", and this checks for file existence (and optionally attribute).